app/scripts/modules/amazon/src/securityGroup/clone/cloneSecurityGroup.controller.js

```diff
@@ -85,6 +85,7 @@ module.exports = angular
         account: credentials,
         accountName: credentials,
         accountId: credentials,
+        securityGroupName: $scope.securityGroup.name,
```

Comment:
instead of `$scope.securityGroup.name` destructure `name` on line 83

```
      const { credentials, name } = $scope.securityGroup;
```
then do 
```suggestion
        securityGroupName: name,
```

Reply:
thank you!